icssc/ZotMeet
# Group Availability Tab

## Description
- Add functionality to view other members' availabilities in the group availability tab.
    - Queries the DB, loads the data via SvelteKit's `load`, and updates the `groupAvailabilities` store.
- Also update drizzle-kit because drizzle-studio only works with the latest version.

## TODOs
- [x] ~~Minimal code to create a meeting.~~ (done in #102)
- [x] Query relevant data from the DB, format it, then update the store(s).
- [x] Update the `groupAvailabilities` store when the user changes their availability.
- [x] Test that everything works.
    - [ ] Write code to create a meeting with multiple members. This is not strictly necessary since we can do this manually while testing.
- [x] Remove sample data generation code.

## Test Plan
- Create a meeting. The personal and group availability stores should be empty.
- Add personal availability and save it. The group availability should reflect without refreshing the page.
- Refreshing the page should load in the most up-to-date group availability.
- Log out of the current user and sign in as another user. Updating personal availability in the same meeting should show the overlapping availability in the groups tab.

## Issues
- Closes #91
